CHANGELOG.md

```diff
@@ -53,6 +53,14 @@
 
 ## breaking changes
 
+* as of v0.13.1-38-gb88c3b84 by default we reject data points with a timestamp far in the future.
+  By default the cutoff is at 10% of the raw retention's TTL, so for example with the default
+  storage schema `1s:35d:10min:7` the cutoff is at `35d*0.1=3.5d`. 
+  The limit can be configured by using the parameter `retention.future-tolerance-ratio`, or it can
+  be completely disabled by using the parameter `retention.enforce-future-tolerance`. 
+  To predict whether Metrictank would drop incoming data points once the enforcement is turned on,
+  the metric `tank.sample-too-far-ahead` can be used, this metric counts the data points which
+  would be dropped if the enforcement were turned on while it is off.
```

Comment:
link to the PR that describes this change

Reply:
I added a link to the PR. I think leaving that short summary in there is better than to remove it though, right? because the initial description of the PR doesn't match the modifications it made in the end after I reacted to all the PR comments, so I think relying on the PR alone as a description of the change could be a bit misleading.

Reply:
yes, leave the summary in the changelog, the changelog should contain the essential info.